xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -263,6 +282,7 @@ class CAddonTimer : public PVR_TIMER
   const std::string m_directory;
   const std::string m_summary;
   const std::string m_seriesLink;
+  std::unique_ptr<PVR_SETTING_KEY_VALUE_PAIR[]> m_customProps;
```

Comment:
Why is this an array as opposed to being a vector? We're in C++ here, so using an array feels wrong

If it's just to make conversion to/from the C API, then fine.

Reply:
> If it's just to make conversion to/from the C API, then fine.

That's is exactly the purpose. Translate C++ to C data structures, to be able to call the C API.

Reply:
Ok, all good.